packages/e2e/test/util.ts

```diff
@@ -30,7 +30,7 @@ const env = getEnv(walletVariables);
 const SECOND = 1000;
 const MINUTE = 60 * SECOND;
 export const TX_TIMEOUT = 7 * MINUTE;
-const SYNC_TIMEOUT = 3 * MINUTE;
+const SYNC_TIMEOUT = 5 * MINUTE;
```

Comment:
Not sure what is reasonable here since we support different networks and have a diversity of wallet sizes. @rhyslbw  @mkazlauskas